### DISCUSSION TITLE
I'm a Verified student but copilot don't connect. It asks me for a membership

### DISCUSSION THREAD

**Discussion by anthxnyR** [community/community#41377] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I was using the copilot and then suddenly it stopped working. I disabled and enabled it since it happened to me not long ago, and then it ask me to activate the account and sent to me to the plan activation but it only appeared with paid options. What can I do?

**Comment by EliasAfara** [comment]
I believe there might be a bug on their end with Github copilot we will have to wait and see.

I have received an email that my free GitHub Copilot access is ending soon.
![image](https://user-images.githubusercontent.com/39487200/206883165-bb52f67e-594c-466b-b0db-306462b9c3e0.png)

So, what i did was I re-applied for the student developer pack and when accepted my copilot subscription was renewed as well. I have received the same message as you.
![image](https://user-images.githubusercontent.com/39487200/206883148-5848b0c3-e2d4-42f5-97ae-9dae92f07feb.png)

I am still not sure why the access will expire on 2023-01-09 though !!

**Reply by beba42** [reply]
I have the same issue

**Reply by saravanabalagi** [reply]
The day has come and I no longer have access to Github CoPilot! :(

**Comment by Akash1134** [comment]
👋 Hi there , Thank you for taking the time to report this issue.

Apologies for the temporary issue on our end that triggered the email and impacted the services. Please retry logging in to your IDE plugin, and Copilot should work as expected.

If you run into any further issues, please reply back in detail with any useful screenshots.

**Reply by amulifts** [reply]
I just had the same issue just now. Can you please look after that.

**Reply by otkrickey** [reply]
@amulifts 
I have created a discussion for today's case. Please share your situation. #43634

**Reply by harshb-fanshawe22** [reply]
I am having similar situation as well. I am a verified student but copilot is not connecting and asking me to pay for the membership. could you please look into that.

**Comment by harshb-fanshawe22** [comment]
I am having similar situation as well. I am a verified student but copilot is not connecting and asking me to pay for the membership. please look into that.

**Reply by otkrickey** [reply]
Check this
Solved: #43634
